src/components/organisms/NewProfileModal/UserProfileModal/UserProfileModal.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from "react";
+import Modal from "react-bootstrap/Modal";
+import { useFirebase } from "react-redux-firebase";
+import { useHistory } from "react-router-dom";
+
+import { IS_BURN } from "secrets";
+
+import { PROFILE_MODAL_EDIT_MODE_TURNING_OFF_DELAY } from "settings";
+
+import { User } from "types/User";
+import { AnyVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { venueLandingUrl } from "utils/url";
+
+import { useShowHide } from "hooks/useShowHide";
+
+import { ProfileModalContent } from "components/organisms/NewProfileModal/ProfileModal/ProfileModalContent";
+import { UserProfileModalContent } from "components/organisms/NewProfileModal/UserProfileModal/UserProfileModalContent";
+
+import "./UserProfileModal.scss";
+
+interface Props {
```

Comment:
```suggestion
export interface UserProfileModal Props {
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/d2dab2490df6b2d756aa2ca0e6d417c4113d7b26